native/src/routes/dashboard/components/__tests__/Dashboard.spec.js

```diff
@@ -0,0 +1,138 @@
+// @flow
+
+import * as React from 'react'
+import TestRenderer from 'react-test-renderer'
+import createNavigationScreenPropMock from '../../../../testing/createNavigationStackPropMock'
+import Dashboard from '../Dashboard'
+import lightTheme from '../../../../modules/theme/constants'
+import CategoriesRouteStateView from '../../../../modules/app/CategoriesRouteStateView'
+import { CityModel } from 'api-client'
+import CategoriesMapModelBuilder from 'api-client/src/testing/CategoriesMapModelBuilder'
+import NavigationTiles from '../../../../modules/common/components/NavigationTiles'
+import buildConfig from '../../../../modules/app/constants/buildConfig'
+
+jest.mock('../../../../modules/common/components/NavigationTiles', () => {
+  const Text = require('react-native').Text
+  return () => <Text>NavigationTiles</Text>
+})
+
+jest.mock('rn-fetch-blob')
+
+describe('Dashboard', () => {
+  const categoriesMapModel = new CategoriesMapModelBuilder('augsburg', 'de').build()
+  const categoryLeaf = categoriesMapModel.toArray().find(category => category.isLeaf(categoriesMapModel))
+  const language = 'de'
+  const navigation = createNavigationScreenPropMock()
+
+  const navigateToPoi = jest.fn()
+  const navigateToCategory = jest.fn()
+  const navigateToEvent = jest.fn()
+  const navigateToInternalLink = jest.fn()
+  const navigateToNews = jest.fn()
+  const navigateToOffers = jest.fn()
+  const theme = lightTheme
+
+  if (!categoryLeaf) {
+    throw Error('There should be a leaf!')
+  }
+
+  const stateView = new CategoriesRouteStateView(
+    categoryLeaf.path,
+    { [categoryLeaf.path]: categoryLeaf },
+    { [categoryLeaf.path]: [] }
+  )
+  const resourceCache = {}
+  const resourceCacheUrl = 'https://localhost:8080'
+  const t = jest.fn(key => key)
+
+  const createCityModel = (events: boolean, offers: boolean, local: boolean, tunews: boolean, pois: boolean) =>
```

Comment:
shouldn't this got to the CityModelsBuilder?

Reply:
There is no api in the builders to set custom values yet, so not possible atm.